### PR TITLE
Fix driving range parsing for hybrid cars

### LINKED ISSUE #69: Invalid carType for Superb iV
2024-09-30 09:50:40.247 ERROR (MainThread) [myskoda.rest_api] Failed to load data from url /v2/vehicle-status/XXXXXXX/driving-range. Return value was '{"carType":"hybrid","totalRangeInKm":677,"primaryEngineRange":{"engineType":"gasoline","currentSoCInPercent":77,"currentFuelLevelInPercent":77,"remainingRangeInKm":670},"secondaryEngineRange":{"engineType":"electric","currentSoCInPercent":25,"remainingRangeInKm":7},"carCapturedTimestamp":"2024-09-30T06:50:07Z"}'
ValueError: 'hybrid' is not a valid EngineType

I have prepared the fix but I have a question:

what do you think, should I reuse EngineType enumeration and add "hybrid" there, or would it be better to introduce new CarType enumeration and repeat all fields from EngineType?
@Prior99 @WebSpider @dvx76

### PR DESCRIPTION
**Pull request by fursov**
Test added.

To fix #69

### CODE REVIEW

**Review by WebSpider — APPROVED** [review]
Smalle request to split tests for engine types. Behaviour may be different.

File: tests/test_driving_range.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+from myskoda.models.driving_range import EngineType
+
+FIXTURES_DIR = Path(__file__).parent.joinpath("fixtures")
+
+
+print(f"__file__ = {__file__}")
+
+
+@pytest.mark.asyncio
+async def test_get_driving_range() -> None:
```
Comment (WebSpider) [inline tests/test_driving_range.py:19]: Can you make this a seperate test for a hybrid engine and a 'mono' engine?

**Review by WebSpider — CHANGES_REQUESTED** [review]

File: tests/test_driving_range.py (hunk 1 above)
Comment (fursov) [reply tests/test_driving_range.py:19]: Would be great if you (or someone else) could send me example of example of the response for mono engine, I will make it part of the test.

Comment (WebSpider) [reply tests/test_driving_range.py:19]: For an electric engine:

```json
{"carType":"electric","totalRangeInKm":139,"primaryEngineRange":{"engineType":"electric","currentSoCInPercent":41,"remainingRangeInKm":139},"carCapturedTimestamp":"2024-10-01T06:54:54Z"}
```

I dont have a mono ICU to contribute at the moment.

Comment (fursov) [reply tests/test_driving_range.py:19]: Which car model is this? We are currently collecting fixtures per car model like enyaq or superb...

**Review by WebSpider — APPROVED** [review]

**Pull request merged**